File: WebService/RTSMS-WebServiceVer1.2.4/Classes/EventTable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Runtime.Serialization;

namespace RTSMSWebService.Classes
{
    [DataContract]
    public class EventTable
    {
        [DataMember]
        public string rssitemID { get; set; }

        [DataMember]
        public String EventIconUrl { get; set; }

        [DataMember]
        public Double Longitude { get; set; }

        [DataMember]
        public Double Lattitude { get; set; }

        [DataMember]
        public String LocationName { get; set; }

        [DataMember]
        public String EventName { get; set; }
 
        [DataMember]
        public String ScreenType { get; set; }

        [DataMember]
        public string Distance { get; set; }

        [DataMember]
        public string ReportedBy { get; set; }

        [DataMember]
        public String ReleaseDate { get; set; }

        [DataMember]
        public String Description { get; set; }

        // if the RSSfeeditem has the weburl else if tweet has the weburl for more detail
        [DataMember]
        public String WebUrl { get; set; }

        [DataMember]
        public bool IsAlert { get; set; }
       
        [DataMember]
        public Int32 AlertLevel { get; set; }
       
        // if future event then the held date
        [DataMember]
        public String HeldDate { get; set; }
        
        // Constructor with Parameters
        public EventTable(string rssitemid,string iconurl,Double longi,Double latti, string locname,string evntname,string screentype,string distance,string reportedby,string releasedate,string description,string  weburl,Boolean isalert,Int32 alertlevel, string helddate)
        {
            this.AlertLevel = alertlevel;
            this.Description = description;
            this.Distance = distance;
            this.EventIconUrl = iconurl;
            this.EventName = evntname;
            this.HeldDate = helddate;
            this.IsAlert = isalert;
            this.Longitude = longi;
            this.Lattitude = latti;
            this.ScreenType = screentype;
            this.LocationName = locname;
            this.ReleaseDate = releasedate;
            this.ReportedBy = reportedby;
            this.rssitemID = rssitemid;
            this.WebUrl = weburl;
        }
    }
}

File: Android/src/com/rtsmsproject/www1/classes/EventTable.java
package com.rtsmsproject.www1.classes;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.text.DateFormat;

import org.json.JSONException;
import org.json.JSONObject;

public class EventTable {

    public String rssitemID;
    public String EventIconUrl;
	public Double Longitude; ;
	public Double Lattitude ;
	public String LocationName ;
	public String EventName ;
	public String ScreenType;
	public String Distance ;
	public String ReportedBy ;
	public String ReleaseDate ;
	public String Description ;
	public String WebUrl ;
	public Boolean IsAlert ;
	public Integer AlertLevel ;
	public String HeldDate ;

	public String getWebUrl()
    {
    	return WebUrl;
    }
    public void setWebUrl(String str) {
    	WebUrl = str;
    }

    public String getRSSItemID()
    {
    	return rssitemID;
    }
    public void setRSSItemID(String str) {
    	rssitemID = str;
    }

	public String getEventIconUrl()
    {
    	return EventIconUrl;
    }
    public void setEventIconUrl(String str) {
    	EventIconUrl = str;
    }

	public Double getLongitude()
    {
    	return Longitude;
    }
    public void setLongitude(Double str) {
    	Longitude = str;
    }

	public Double getLattitude()
    {
    	return Lattitude;
    }
    public void setLattitudeD(Double str) {
    	Lattitude = str;
    }

	public String getLocationName()
    {
    	return LocationName;
    }
    public void setLocationName(String str) {
    	LocationName = str;
    }

	public String getEventName()
    {
    	return EventName;
    }
    public void setEventName(String str) {
    	EventName = str;
    }

	public String getScreenType()
    {
    	return ScreenType;
    }
    public void setScreenType(String str) {
    	ScreenType = str;
    }

	public String getDistance()
    {
    	return Distance;
    }
    public void setDistance(String str) {
    	Distance = str;
    }

    public String getReportedBy()
    {
    	return ReportedBy;
    }
    public void setReportedBy(String str) {
    	ReportedBy = str;
    }

    public String getReleaseDate()
    {
       // DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
    	//return dateFormat.format(ReleaseDate);
    	return ReleaseDate;
    }
    public void setReleaseDate(String str) {
    	ReleaseDate = str;
    }

    public String getDescription()
    {
    	return Description;
    }
    public void setDescription(String str) {
    	Description = str;
    }

    public Boolean getIsAlert()
    {
    	return IsAlert;
    }
    public void setIsAlert(Boolean str) {
    	IsAlert = str;
    }

    public Integer getAlertLevel()
    {
    	return AlertLevel;
    }
    public void setAlertLevel(Integer str) {
    	AlertLevel = str;
    }

    public String getHeldDate()
    {
       // DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
    	//return dateFormat.format(HeldDate);
    	return HeldDate;
    }
    public void setHeldDate(String str) {
    	HeldDate = str;
    }
    
    
}
